backend/app/controllers/tasks.js

```diff
@@ -120,10 +101,11 @@ module.exports = {
                 )
                 .where(Task.id.equals(req.params.id))
             );
+
             if (!_.first(task)) {
-                throw new HttpError(403, 'Not found');
+                throw new HttpError(403, 'Item with that ID does not exist.');
```

Comment:
Do you want to pass an error code instead, since there is an error codes file, or are we trying to avoid that now?
Using a code instead of a string keeps things DRY.